frontend/src/App.jsx

```diff
@@ -10,6 +10,7 @@ import Login from "./Pages/Login";
 import PostDetail from "./Pages/PostDetail";
 import Profile from "./Pages/Profile";
 import SearchResults from "./Pages/SearchResults";
+import Copyright from "./Layout/Copyright";
```

Comment:
Die Datei "Layout.jsx" ist nicht in den Commits enthalten. Kannst du die noch hinzufügen? Gibt es einen Grund, warum die Datei so spezifisch "Copyright" heißt, und nicht "Footer" oder so? Also sind wir uns sicher, dass da immer nur ein Copyright drinstehen wird?